src/server/PostManager.js

```diff
@@ -1,6 +1,34 @@
+import db from '../server/database';
+import {KeyUtils, ENTITY} from '../server/key-utils';
 
 export default class PostManager {
-  constructor() {
-
+  static addCreatedPost(entity, itemKey, userKey, timeHash, cb) {
+    if (entity !== ENTITY.ITEM && entity !== ENTITY.IMAGE) {
+      return cb(new Error(('invalid entity'), null));
+    }
+    const idxKey = KeyUtils.getIdxKey(ENTITY.CREATED_POST, timeHash, userKey);
+    const idxPost = {
+      entity: entity,
+      key: itemKey
+    };
+    return db.put(idxKey, idxPost, (err) => {
+      if (err) {
+        return cb(new Error('error while putting in DB'), null);
+      }
+      return cb(null, idxKey);
+    });
```

Comment:
We don't have to make createdPost.
Cause indexing createdPost has item key so we can make just indexing createdPost to searching.